### ISSUE TITLE
Merge Flake8 Lint with native Brackets linters

### ISSUE THREAD

**Issue by tomek-he-him** [marcoceppi/brackets-flake8#3]
This is a feature suggestion.

It would be great when Flake8 linter could behave like the other linters out there (e.g. JSlint or CoffeeLint).

Important points:
- toggle on/off and see clean/dirty status on the status bar
- move cursor to appropriate line upon clicking the error

**Comment by marcoceppi** [comment]
I agree, I need to read up on the native bracket linting interface, but this should just be toggled for any Python file and not do it's own determining if it should lint
